src/guides/v2.3/comp-mgr/patching.md

```diff
@@ -75,14 +79,207 @@ index c8a6fef58d31..7d01c195791e 100644
 
 ## Applying patches
 
-There are two ways to apply patches:
+You can apply patches using any of the following methods:
 
--  Using the command line
--  Using Composer
+-  Magento Quality Patch (MQP) package
+-  Command line
+-  Composer
 
 {:.bs-callout-info}
 To apply a patch to a {{site.data.var.ece}} project, see [Apply patches][].
 
+### Magento Quality Patch (MQP) package {#mqp}
+
+The [MQP package][] delivers individual patches developed by Magento and allows you to apply, revert, and view general information about all individual patches that are available for the installed version of {{ site.data.var.ee }} or {{ site.data.var.ce }}.
+
+{:.bs-callout-warning}
+We do not recommend using the MQP package to apply large numbers of patches because it increases the complexity of your code, which makes upgrading to a new version of Magento more difficult.
+
```

Comment:
#### Installation prerequisites

MQP requires installed [Git](https://git-scm.com/) to perform operations with patches. See [instructions](https://github.com/git-guides/install-git) on how to install [Git](https://git-scm.com/) on your instance.